compiler/src/iree/compiler/InputConversion/MHLO/ConvertCollectiveOps.cpp

```diff
@@ -335,6 +335,162 @@ struct AllReduceOpConversion : public OpConversionPattern<mhlo::AllReduceOp> {
   }
 };
 
+Value EmitTranspose(ConversionPatternRewriter &rewriter, const Location &loc,
```

Comment:
style nit: `emitTranspose` (llvm uses camelCase for standalone functions)
(here and below)